src/main/java/uk/gov/pay/directdebit/payments/dao/PaymentViewDao.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.pay.directdebit.payments.dao;
+
+import org.jdbi.v3.core.Jdbi;
+import uk.gov.pay.directdebit.payments.dao.mapper.PaymentViewMapper;
+import uk.gov.pay.directdebit.payments.model.PaymentView;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class PaymentViewDao {
+
+    private final Jdbi jdbi;
+
+    private StringBuilder sb = new StringBuilder(
```

Comment:
You don't need a string builder, this can just be a plain string

Reply:
Yes, I'm aware that the compiler turns this String = +String into a StringBuilder under the hood. What I've done looks better and confirms to clean coding principles.